crl/updater/updater.go

```diff
@@ -467,17 +480,24 @@ func (cu *crlUpdater) tickShard(ctx context.Context, atTime time.Time, issuerNam
 // there is a buffer of at least one whole chunk width between the actual
 // furthest-future expiration (generally atTime+90d) and the right-hand edge of
 // the window (atTime+lookforwardPeriod).
-func (cu *crlUpdater) getShardBoundaries(atTime time.Time, shardIdx int) (time.Time, time.Time) {
+func (cu *crlUpdater) getShardBoundaries(atTime time.Time, shardIdx int) (time.Time, time.Time, error) {
 	// Ensure that the given shard index falls within the space of acceptable indices.
 	shardIdx = shardIdx % cu.numShards
 
 	// Compute the width of the full window.
 	windowWidth := cu.lookbackPeriod + cu.lookforwardPeriod
+
+	// Compute the amount of time between the current time and the anchor time.
+	timeSinceAnchor := atTime.Sub(startTime)
```

Comment:
The comments say anchor, but the variable is `startTime` and the comment on the definition of `startTime` doesn't say anything about anchor time. It would be nice to keep those consistent, and also update the big block comment on getShardBoundaries to talk about the "anchor time".

Reply:
Standardized to use "anchor time" everywhere.